### DISCUSSION TITLE
cmi.command.vanishdefault.X (for ex. damageToEntity)

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#4636] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, could you add way to set specific permission for default vanish settings please? So we could set there:
```
    damageToEntity: false
    playerDamage: false
    itemPickup: false
    mobAggro: false
    interaction: false
    noisyChest: false
    informOnLeave: false
    informOnJoin: false
    nightVision: true
    bossbar: true
    afkcommands: false
    PrivateMessages: false
    relogDisable: false
    noMessages: false
    fakeJoinLeave: false
    mobSpawning: false
```
by permissions.

### Basic explanation as to why this feature is needed and/or useful to have: ###
Would be useful when using /vanish for VIP players, but also for moderators etc. These VIP players  will be really high now cause they don't get damage by monsters etc.... but when I disable it as default, then all of my staff have to change settings, and they could forget on it and they will be seen then.

**Comment by Zrips** [comment]
In general giving vanish to regular players just invites additional headache. But if you want to, then you might simply set those defaults to fit vip players while staff can always adjust them, once, with vanishedit command to fit their needs, after which their setup will be saved and it will be one time thing.

**Comment by bobhenl** [comment]
Yeah I know they could do it this way, just though that these default settings could be set via permissions too, so they wouldn't have to edit it personally.
